### PR TITLE
CurrentSite: fix loading state.

### PR DESCRIPTION
**Pull request by mtias**
Before:
image

After:
image

### CODE REVIEW

File: client/my-sites/current-site/index.jsx [hunk 1]
```diff
@@ -156,11 +156,11 @@ module.exports = React.createClass( {
 		if ( ! this.props.sites.initialized ) {
 			return (
 				<Card className="current-site is-loading">
-					<div className="site">
 					{ hasOneSite
 						? this.addNewWordPressButton()
-						: <span className="current-site__switch-sites" />
+						: <span className="current-site__switch-sites">&nbsp;</span>
```
Comment (aduth) [inline client/my-sites/current-site/index.jsx]: Wouldn't react render this as plain-text `&nbsp;` ? Should this be a space charater ` ` instead?

**Comment by rralian** [comment]
:shipit:

File: client/my-sites/current-site/index.jsx (hunk 1 above)
Comment (mtias) [inline client/my-sites/current-site/index.jsx]: It's rendering it fine for me.

Comment (mtias) [inline client/my-sites/current-site/index.jsx]: @aduth you can insert HTML entities within literal text in JSX, the problem is if you wrap it in `{ }`

Comment (rralian) [inline client/my-sites/current-site/index.jsx]: Huh... I wasn't sure, disabled javascript during this state to inspect to see what happens, and it renders as a space.
space

Comment (aduth) [inline client/my-sites/current-site/index.jsx]: Yep, upon double-checking the docs, @mtias is right and this only affects dynamic content - https://facebook.github.io/react/docs/jsx-gotchas.html#html-entities

**Pull request merged**